src/tree/SlotTreeItemBase.ts

```diff
@@ -3,7 +3,9 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { AppSettingsTreeItem, AppSettingTreeItem, deleteSite, ISiteTreeRoot, SiteClient } from 'vscode-azureappservice';
+// tslint:disable-next-line:no-submodule-imports
+import * as WebSiteModels from 'azure-arm-website/lib/models';
```

Comment:
nit: `import { WebSiteManagementModels } from 'azure-arm-website';` without disabling the lint rule